### DISCUSSION TITLE
Text detection for BILLS

### DISCUSSION THREAD

**Discussion by navaneth99** [ultralytics/ultralytics#5290] [General]
Can we train Yolov8 to recognise text? data includes receipts from internet purchases Can Yolov8 in these circumstances detect characters? If we train with enough data, can it detect.dataset the bill pattern is always shifting If we train the yolov8 model on, for example, 100 patterns of various bills, will it be able to recognise the characters?Can a Yolov8 be trained to handle scenarios like these?

**Comment by glenn-jocher** [comment]
@navaneth99 absolutely, YOLOv8 can be trained to recognize text on bills and receipts, even with varying patterns. With a sufficiently diverse and representative dataset that includes various bill formats, the model can learn to detect and recognize characters. The key is to ensure your dataset covers the range of variations you expect the model to handle in real-world scenarios. Once trained, YOLOv8 should generalize to detect text patterns on unseen bills, provided the training data was comprehensive. Remember to annotate your data accurately for the best results! 😊🧾🔍

**Reply by navaneth99** [reply]
@glenn-jocher  Thanks for the reply

**Reply by glenn-jocher** [reply]
@navaneth99 you're welcome! If you have any more questions or need further assistance as you work on your text detection project, feel free to reach out. Good luck with your YOLOv8 training, and happy detecting! 😄🚀
